components/DownloadModal/DownloadModal.scss

```diff
@@ -37,7 +37,8 @@
       width: 112px;
       height: 112px;
       margin-right: 20px;
-      background: url('/static/nos-app-icon.svg');
+      background: url('/static/nos-app-icon.png');
```

Comment:
Looks like there's a filename mismatch that works on Windows, but won't work on the web server. The filename referenced is lowercase here, but contains capital letters in the actual filename.